src/util.js

```diff
@@ -27,3 +27,44 @@ export function isString(value) {
 export function isFunction(value) {
   return typeof value === 'function';
 }
+
+/**
+ *  This Function Creates element based on type passed
+ *  currently limited to Link and script element
+ *
+ * @param {*} url
+ * @param {*} type
+ * @returns
+ */
+export function createElement(url, type) {
+  let element = null;
+
+  if (type === 'text/css') {
+    element = document.createElement('link');
+    element.rel = 'stylesheet';
+    element.type = 'text/css';
+    element.href = url;
+  } else if (type === 'text/javascript') {
```

Comment:
You can return early and this can help you omit `else if` - like this:
```suggestion
    return element;
  }
  
  if (type === 'text/javascript') {
```